OpenIPSL/Interfaces.mo

```diff
@@ -2,7 +2,7 @@ within OpenIPSL;
 package Interfaces
```

Comment:
Yes you are right the external packages would need updating. I haven't started with the tutorial itself yet.